indra/sources/biopax/processor.py

```diff
@@ -615,13 +615,24 @@ def _get_processed_xrefs(bpe: bp.PhysicalEntity):
         xrefs = defaultdict(set)
         if primary_ns and primary_id:
             xrefs[primary_ns].add(primary_id)
+            # In the case of CHEBI, we return directly, no further processing
+            # of xrefs since xrefs tend to contain other related CHEBI IDs
+            # representing other chemicals
+            if primary_ns == 'CHEBI':
+                return xrefs
 
         for xref in entref.xref:
             if not xref.db:
                 continue
             xref_db_ns = xref_ns_map.get(xref.db.lower())
             if not xref_db_ns:
                 continue
+            # This is the "see also" relation which points to related but
+            # not exact xrefs that we can skip here
+            if isinstance(xref, bp.RelationshipXref) and \
+                xref.relationship_type and xref.relationship_type.uid == \
+                    'http://identifiers.org/psimi/MI:0361':
```

Comment:
are they always written this way? this is a goofy identifiers.org URL since it uses both the `namespace/id` syntax in combination with the namespace embedded in LUI. Maybe also test for `xref.relationship_type.uid in {'http://identifiers.org/psimi/MI:0361', 'http://identifiers.org/MI:0361'}`

Reply:
Yes, that's how it appears in the PC12 OWL. It's also weird becuase `psimi` isn't a valid identifiers.org prefix. The correct "old style" URL would actually be http://identifiers.org/mi/MI:0361, and the correct "new style" URL would be the one you added, http://identifiers.org/MI:0361, but neither of those are actually used!

Reply:
I added a more general solution